docs/reference/api-v1.1.rst

```diff
@@ -16,6 +16,8 @@ What's New
 
 **New!** All controller responses now return the ``X_DEIS_PLATFORM_VERSION`` header.
 
+**New!** Users see a 404 response when modifying applications they are unauthorized to see.
+
```

Comment:
Should we also update all the example requests in this doc to say `X_DEIS_API_VERSION: 1.1.2`? Looks like we didn't do that for the 1.1.1 API change, but I think that would be consistent. I've been updating the `X_DEIS_PLATFORM_VERSION:` in the examples with each release.